tests/perf/pubsub_publish_http/test.js

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright 2022 The Dapr Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import http from "k6/http";
+import { check } from "k6";
+import crypto from "k6/crypto";
+
+const KB = 1000
```

Comment:
1000 or 1024?

Reply:
well it depends on the [base system used](https://en.wikipedia.org/wiki/Kilobyte), but I think we can assume base 2 and use 1024

Reply:
Yeah, we normally use base 2 throughout Dapr.